pandas/tests/groupby/test_transform.py

```diff
@@ -317,6 +317,33 @@ def test_dispatch_transform(tsframe):
     tm.assert_frame_equal(filled, expected)
 
 
+@pytest.mark.parametrize("func", ["fillna", "ffill", "bfill", "shift", "cumsum"])
```

Comment:
We have a fixture already called transformation_func in the GroupBy conftest you can use here instead

Reply:
Do you mean we should test them all here? We may have quite some overlapping coverage with existing test cases though